pennylane/ops/op_math/exp.py

```diff
@@ -181,6 +184,11 @@ def has_decomposition(self):
                 )
         return math.real(self.coeff) == 0 and is_pauli_word(self.base)
 
+    @property
+    def inverse(self):
+        """Setting inverse not defined for Exp, so inverse is always False"""
+        return False
```

Comment:
Why was this needed?

Reply:
`qml.generator(op)` checks it, so if we want it to behave like other classes in terms of accessing the generator, it needs to have this property until inverse is removed everywhere

Reply:
We could update `generator` to use `getattr(op, "inverse", False)` instead of `op.inverse`.

Reply:
Is it correct that if we change it so `Exp` inherits from `Operation`, we need to overwrite this (and a couple other things) anyway? Since we don't want to have inversion on `Exp`.